src/pre_defined_classes/programs_with_changing_behaviour/hard/q2/MainTest.java

```diff
@@ -0,0 +1,36 @@
+package pre_defined_classes.programs_with_changing_behaviour.hard.q2;
+
+import global.BaseRandomTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.RandomInteger;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class MainTest extends BaseRandomTest {
+    //Parsons with Distractors
```

Comment:
Maybe add a space before the comment just for perpetuity and consistency

Reply:
Ok I did this